mkt/webpay/webpay_jwt.py

```diff
@@ -18,7 +19,14 @@
 
 
 def get_product_jwt(product, contribution):
-    """Prepare a JWT for paid products to pass into navigator.pay()"""
+    """
+    Prepare a JWT describing the item about to be purchased when
+    working with navigator.mozPay()
+
+    See the MDN docs for details on the JWT fields:
+    https://developer.mozilla.org/en-US/Marketplace/Monetization
```

Comment:
Does pyflakes get mad if you leave this URL on one line? I feel like it should be okay to leave this as one line in a docstring.